### PR TITLE
Fix #4760: Correct guest entry points on loading scenario

### LINKED ISSUE #4760: Great Wall of China - Guests are spawning at a point where you cannot enter
**OS:** Windows 10
**Version:** 0.0.5
**Commit/Build:** c9a1ce5

In my Great Wall of China Save (from Wacky Worlds), there exist guests who are trying spawning and trying to enter a little to the right of the actual entrance. I'm not sure why this is.

- [N] Reproducible in RCT2 (vanilla)?
- [N] Multiplayer?

**Save game:**
[Great Wall of China.zip](https://github.com/OpenRCT2/OpenRCT2/files/573851/Great.Wall.of.China.zip)

### PR DESCRIPTION
**Pull request by Gymnasiast**

### CODE REVIEW

File: src/openrct2/rct2/S6Importer.cpp [hunk 1]
```diff
@@ -701,6 +703,36 @@ class S6Importer final : public IParkImporter
...
         game_init_all(_s6.map_size);
     }
+
+    /**
+     * Clears incorrectly set guest entry points.
+     */
+    void FixEntryPoints()
+    {
+        // While we probably could fix both saves and scenarios, I want to avoid touching saves where the user
+        // might already have fixed this, or just happens to share a name.
```
Comment (IntelOrca) [inline src/openrct2/rct2/S6Importer.cpp]: If you write in 1st person, you should probably include a signature.

**Comment by Broxzier** [comment]
Wouldn't it be better to check if a park entrance can be reached from an entry point, instead of comparing known filenames?

**Comment by Gymnasiast** [comment]
@Broxzier Like I just said in the chat, no. Not without large complexity and enormous side effects.

File: src/openrct2/rct2/S6Importer.cpp (hunk 1 above)
Comment (Gymnasiast) [reply src/openrct2/rct2/S6Importer.cpp]: I can write it in third person, if you can get behind my reasoning here.

Comment (IntelOrca) [reply src/openrct2/rct2/S6Importer.cpp]: ok

**Comment by Broxzier** [comment]
Guests somehow do not spawn at at that point in Rio Carnival though, so I figured there's a check being done somewhere already.

**Comment by Gymnasiast** [comment]
> Guests somehow do not spawn at at that point in Rio Carnival though, so I figured there's a check being done somewhere already.

They do. How would people have noticed otherwise?

**Pull request merged**